pkg/controller/admissionchecks/provisioning/controller.go

```diff
@@ -302,6 +303,10 @@ func (c *Controller) syncOwnedProvisionRequest(ctx context.Context, wl *kueue.Wo
 			}
 
 			if err := c.client.Create(ctx, req); err != nil {
+				ac.Message = fmt.Sprintf("Error creating ProvisioningRequest %q: %v", requestName, err)
```

Comment:
@IrvingMg it just occurred to me - could you please follow up to use the TruncateConditionMessage, and TruncateConditionMessage helpers to make sure the message is not too long? 

It shouldn't be a problem in most cases, but since the message may come from custom webhooks it may be arbitrarily long.

Reply:
cc @mbobrovskyi